src/CellData/CellFields.jl

```diff
@@ -278,7 +278,8 @@ function _point_to_cell!(cache, x::Point)
   searchmethod, kdtree, vertex_to_cells, cell_to_ctype, ctype_to_polytope, cell_map, table_cache = cache
 
   # Loop over the first m.num_nearest_vertex
-  for (id,dist) in zip(knn(kdtree, SVector(Tuple(x)), searchmethod.num_nearest_vertices, true)...)
+  x_val = ForwardDiff.value.(x.data)
```

Comment:
Could we maybe add 

```julia
@inline function ForwardDiff.value(x::MultiValue{S,<:ForwardDiff.Dual}) where S
  VT = change_eltype(x,ForwardDiff.valtype(eltype(x)))
  data = map(ForwardDiff.value,x.data)
  return VT(data)
end
```

to the `TensorValues` modules instead and then 

```julia
x_val = ForwardDiff.value(x)
```

I think it is more generic and reusable. It also defaults to doing nothing whenever we do not have dual numbers.

Reply:
changes made